### PR TITLE
feat: org permissions UI

### PR DESCRIPTION
**Pull request by nathaliabj**
### Pre requirements:
- [ ] I have followed (at least) the [PR section of the contributing guide](https://github.com/lightdash/lightdash/blob/main/.github/CONTRIBUTING.md#sending-a-pull-request).

### Reference:
Closes: 

### Scope of the changes (select all that apply):
- [ ] Frontend  
- [ ] Backend  
- [ ] Common  
- [ ] E2E tests  
- [ ] Docs  
- [ ] CI/CD

### CODE REVIEW

**Comment by nathaliabj** [comment]
Thinking about what will the non admin users see on the `User management` tab  @PriPatel suggests to remove this option all together for them. 

options are:
- disable this tab when they are non admin
- remove the tab all together
- show a screen saying that they don't have access instead of showing the list of users

@owlas @ZeRego @PriPatel any thoughts? 💭

**Comment by owlas** [comment]
@nathaliabj sounds good! We can remove the entry in the sidebar if they don't have permission

File: packages/frontend/src/components/UserSettingsModal/OrganisationPanel/index.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ const updateOrgQuery = async (data: { organizationName: string }) =>
         body: JSON.stringify(data),
     });
 
-const OrganizationPanel: FC = () => {
+const OrganisationPanel: FC = () => {
```
Comment (owlas) [inline packages/frontend/src/components/UserSettingsModal/OrganisationPanel/index.tsx:15]: We usually use US spelling!

**Review by owlas — CHANGES_REQUESTED** [review]
Looks good! Just one thing, Organizations changed from US to British spelling

**Review by owlas — APPROVED** [review]
lgtm! 🔥

**Pull request merged**